src/components/panel/panel.js

```diff
@@ -743,6 +743,12 @@ function MdPanelService($rootElement, $rootScope, $injector, $window) {
    * @type {enum}
    */
   this.yPosition = MdPanelPosition.yPosition;
+
+  /**
+   * Margin between the edges of the panel and the viewport
+   * @type {number}
+   */
+  this.viewportMargin = MdPanelPosition.viewportMargin;
```

Comment:
The main reason was to expose it to the unit tests. It can also be a global variable there that can be easily changed if tests start failing because of the wrong margin.